hack/lib/serverless.bash

```diff
@@ -406,6 +406,10 @@ EOF
     yq write --inplace "$knativekafka_cr" spec.high-availability.replicas 1
   fi
 
+  if [[ $ENABLE_KEDA == "true" ]]; then
+    yq write --inplace "$knativekafka_cr" 'spec.config-kafka-features."controller.autoscaler"' "enabled"
```

Comment:
note: make sure that this is compatible with yq 3.4.*

Reply:
this was tested locally with:
```
yq -V     
yq version 3.4.1
```
would that work @pierDipi ?